src/wam2layers/utils/profiling.py

```diff
@@ -129,12 +135,12 @@ def track_stability_correction(self, fy_corrected, fy_abs, config, coords, t):
         timestamp = t.strftime("%Y%m%d-%H%M%S")
         filename = debug_dir / f"stability_correction_{timestamp}.nc"
 
-        ncfile = xr.DataArray(correction, coords=coords, name="correction")
-        ncfile.to_netcdf(filename)
+        correction.name = "correction"
+        correction.to_netcdf(filename)
 
         logger.warn(
             f"Stability correction applied to {corrected_percent:.1f}% of "
             f"grid. Average correction was {correction.mean():.1f}, "
-            f"maximum correction was {correction.max()}. The total "
+            f"maximum correction was {correction.max():.1f}. The total "
```

Comment:
Not sure if we should limit the absolute value to 1 decimal place, perhaps use scientific notation?

Reply:
I guess we can do that. I mostly copied the formatting from the previous lines so it would format it as a float and not display the xarray object.

Reply:
Then we could also use `.item()` or `.values()`